README.adoc

```diff
@@ -815,14 +819,10 @@ deployments, systems, and routes:
 oc delete -f deploy.yaml
 ```
 
-To completely delete your Minishift VM, cluster, and all associated files run the following command:
-
-[role=command]
-```
-minishift delete
-```
+To completely delete your Minishift VM, cluster, and all associated files, refer to the official 
+https://docs.okd.io/latest/minishift/getting-started/uninstalling.html[Uninstalling Minishift^] documentation.
 
-To revert your Docker daemon back to its default settings, simply reboot your system.
+To revert your Docker daemon back to its default settings, simply close the shell session.
```

Comment:
```suggestion
To revert your Minishift Docker daemon back to its default settings, simply close the shell session.
```

Reply:
The guide asks the user to allow their system to use Minishift's daemon, so technically we're not reverting the Minishift daemon back to the original settings, instead, we're disconnecting our console from the built-in Minishift Docker daemon and allowing our system to revert back to using our Docker daemon.

Reply:
Then, let us explain that appropriately.  Both before and after.  It wasn't clear still what we are asking the users to do.  Your explanation here is much better than what we have.